.github/workflows/ci-linux-incremental.yml

```diff
@@ -18,12 +18,11 @@ name: CI Linux incremental
 on:
   pull_request:
     types:
-      # Defaults
-      - opened
-      - synchronize
-      - reopened
```

Comment:
Why this change?

Reply:
Isn't it sufficient to run the workflow when you add the label?

Reply:
No, that's not how it's intended.

Reply:
okay, reverted.

Reply:
I think we should rather remove "labeled" filter. This would trigger the workflow whenever any new label is added such as "needs review". No?